share/brewkit/fix-machos.rb

```diff
@@ -1,8 +1,6 @@
-#!/usr/bin/env ruby
+#!pkgx +gem ruby
```

Comment:
generally what we use (for sanity) is `#!/usr/bin/env -S pkgx +gem ruby` as the preferred method.